app/org/maproulette/models/Task.scala

```diff
@@ -40,9 +40,15 @@ case class Task(override val id: Long,
                 geometries: String,
                 suggestedFix: Option[String] = None,
                 status: Option[Int] = None,
-                priority: Int = Challenge.PRIORITY_HIGH,
+                mappedOn: Option[DateTime] = None,
```

Comment:
Should this maybe be stored in it's own table. Like. TaskReview table that just references the Task table and can be retrieved when required?